scripts/exglobal_archive.py

```diff
@@ -35,12 +35,12 @@ def main():
 
     archive_dict = AttrDict()
     for key in keys:
-        archive_dict[key] = archive.task_config[key]
+        archive_dict[key] = archive.task_config[key, "Undefined"]
 
     # Also import all COMIN* and COMOUT* directory and template variables
     for key in archive.task_config.keys():
         if key.startswith("COMIN_") or key.startswith("COMOUT_"):
-            archive_dict[key] = archive.task_config[key]
+            archive_dict[key] = archive.task_config.get[key]
```

Comment:
You'll want to use the `get` method in both cases and provide defaults:

```suggestion
        archive_dict[key] = archive.task_config.get(key, "Undefined")

    # Also import all COMIN* and COMOUT* directory and template variables
    for key in archive.task_config.keys():
        if key.startswith("COMIN_") or key.startswith("COMOUT_"):
            archive_dict[key] = archive.task_config[key]
            archive_dict[key] = archive.task_config.get(key, 'Undefined')
```